pkg/controller/servicemesh/memberroll/controller.go

```diff
@@ -396,13 +400,25 @@ func (r *MemberRollReconciler) reconcileObject(ctx context.Context, roll *maistr
 }
 
 func isExcludedNamespace(ns string) bool {
+	// keep this in sync with getExcludedNamespaces()
```

Comment:
perhaps this one could be refactored to sth like `return ns in  getExcludedNamespaces()`

Reply:
I've now done this, since getExcludedNamespaces() returns regex patterns.